tests/regrtest_data/len_crash_tester/test_file.py

```diff
@@ -0,0 +1,18 @@
+"""Module docstring"""
+
+
+class MyClass:
+    """Class to test crash reported in:
+    https://github.com/PyCQA/pylint/issues/5244
+    """
+
+    def __init__(self, array):
+        self.array = array
+
+    def __len__(self):
+        """Initially reported crash crashed on this function def"""
+        return len(self.array)
```

Comment:
This doesn't test anything anymore. Apparent as all tests pass without the fix.
Either revert the module name change or use the example for the code review (with `lambda`). This can even be added as a normal functional test as it doesn't depend on the module name.
```py
class MyClass:
    def some_func(self):
        return lambda: 42

    def __len__(self):
        return len(self.some_func())
```